peer/brontide.go

```diff
@@ -3878,3 +3865,33 @@ func (p *Brontide) handleNewPendingChannel(req *newChannelMsg) {
 	p.addedChannels[chanID] = struct{}{}
 	p.activeChanMtx.Unlock()
 }
+
+// sendLinkUpdateMsg sends a message that updates the channel to the
+// channel's message stream.
+func (p *Brontide) sendLinkUpdateMsg(msg lnwire.Message) {
+	linkUpdateMsg, ok := msg.(LinkUpdater)
+	if !ok {
+		return
+	}
+
+	targetChan := linkUpdateMsg.TargetChanID()
```

Comment:
Since we now determine `targetChan` here, I think we can remove the associated variable in `readHandler`.

Reply:
not sure I follow, what do you mean by "the associated variable"?

Reply:
I'm not sure anymore -- it looks like the code I was referring to has changed, but the force push makes it hard to figure out what's different.